src/API/src/export/export.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Inject, Injectable, Logger } from '@nestjs/common';
+import { OccurrenceService } from 'src/db/occurrence/occurrence.service';
+import { flattenOccurrenceRepoObject } from './utils/allDataCsvCreation';
+import { occurrenceMapper } from './utils/occurrenceMapper';
+const fs = require('fs');
+
+@Injectable()
+export class ExportService {
+  constructor(
+    @Inject(OccurrenceService)
+    private readonly occurrenceService: OccurrenceService,
+    private logger: Logger,
+  ) {}
+
+  async exportOccurrenceDbtoCsvFormat() {
+    const dbData = await this.occurrenceService.findAll();
```

Comment:
Might want to stick this in the try block too

Reply:
Did this happen?

Reply:
I must have missed this one. It should now be within the try block